resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,22 @@
+<x-prompt-layout
+    :page-title="__('Login')"
+>
+    <x-slot name="header">
+        <h1 class="mb-0">{{ __('Sign in to RetroAchievements') }}</h1>
+    </x-slot>
+
+    <x-form :action="route('login')">
+        <x-input.text attribute="User" required />
+        <x-input.password attribute="password" required />
+        <x-input.checkbox attribute="remember" :label="__('Remember Me')" />
+        <x-button type="submit" class="w-full text-center py-2 mb-4">{{ __('Sign in') }}</x-button>
+        <div class="flex flex-col justify-between">
+            <a class="btn btn-link" href="{{ url('resetPassword.php') }}">
+                {{ __('Forgot your password?') }}
+            </a>
+            <a class="btn btn-link" href="{{ url('createaccount.php') }}">
+                {{ __('Sign up') }}
+            </a>
+        </div>
+    </x-form>
```

Comment:
Failed login reports the error twice.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/3db5ff7c-f3e8-4298-8303-874260d441fd)

Reply:
Successful login always returns to the home page. Before this PR, the login would remain on the current page. I'm not sure if it's a deal-breaker, but I often notice that I'm not logged in when trying to access the Dev dropdown on a game page and seeing it's not there. Being able to login without losing my context was a nice feature. I understand that having an intermediate page complicates things.

Reply:
It would be ideal if there was some kind of callbackUrl or redirectUrl that after login is successful, sends the user back to that URL. Could be passed to the page as a query param.

Reply:
Laravel supports that put of the box as "intended_url" session parameter and should redirect back automatically.
I'll look into why that's not happening.

Reply:
Seems that redirect after login has to be done by hand with fortify 🤔 
Fixed in https://github.com/RetroAchievements/RAWeb/pull/1831/commits/704d4bc4e40ba259b41c9bad114c1796287b6be9

Removed global messages from prompt layout in https://github.com/RetroAchievements/RAWeb/pull/1831/commits/e27de4cbf0818b3ff02e6961616f5a33f26e6c22